massgov/mayflower
# DP-26237 [SI-a11y] Collapsible content label

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

- Correct the markup, it's invalid that a heading is nested to a button.
- Clean up confusing aria-label content
- Add context for the item count to make sense to screen reader users
- Hide "+/-" accordion state icon from screen readers

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-26237)
- [Github issue]()

## Steps to Test


1. Go to /patternlab/patterns/03-organisms-by-author-collapsible-content-extended/03-organisms-by-author-collapsible-content-extended.html
2. In inspect, find:
    - `button` is in a heading.
    - "+" symbol on the right in the button is not css content.
    - No `aria-label` on `button`s.
3. Check `aria-expanded` gets updated as open/close the accordions.
4. No visual change.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
